tests/playwright/utilities/terminal.js

```diff
@@ -0,0 +1,36 @@
+import { execSync } from 'node:child_process';
+import path from 'path';
+
+class Terminal {
+	/*
+     * Activate or Deactivate experiments
+     * @param action activate | deactivate
+     * @param experiments comma separated list of experiments. e.g. container,nested-elements,'grid container'
+     */
+	async experiments( action, experiments ) {
+		const wpPath = await this.getWordpressInstallPath();
+
+		const wpEnv = `npx wp-env run cli "bash -c 'wp elementor experiments ${ action } ${ experiments }'"`;
+		const local = `bash -c "wp elementor experiments ${ action } ${ experiments } --path=${ wpPath }"`;
+
+		const result = await this.executeTerminal( process.env.CI ? wpEnv : local );
+		console.log( result );
```

Comment:
```suggestion
```

Reply:
@louiswol94 shows any errors in terminal. My thinking is this would be useful locally and on CI if you specified a wrong experiment name in a Playwright test. It's very hard to test things that fail silently. Updates to the Elementor cli now shows which experiment is invalid in the output not just an error. But the process still either fails or passes so we don't introduce side effects and returns that result.

![image](https://github.com/elementor/elementor/assets/130742176/445b3381-7e99-47ba-8b11-345b2b8fe30f)

![image](https://github.com/elementor/elementor/assets/130742176/3874507d-097b-4bad-8e22-85cd7e5e8a3a)

Reply:
In that case maybe add it comment to it, or extra it to a function like `errorLogging()` for clarity during future development.